worth2/main/generic/models.py

```diff
@@ -57,6 +54,12 @@ def unlocked(self, user):
 
         return True
 
+    def pageblock(self):
+        return self.pageblocks.first()
+
+    # TODO: I'd like to have all the following methods be inherited
```

Comment:
An abstract base class might be interesting. https://docs.python.org/2/library/abc.html